src/server/readpool/config.rs

```diff
@@ -134,13 +80,113 @@ impl Config {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Default)]
+#[serde(default)]
+#[serde(rename_all = "kebab-case")]
+/// A helper struct to handle deserialization default values.
+pub struct SerdeConfigHelper {
+    pub high_concurrency: Option<usize>,
```

Comment:
Strange and ugly to use Option here. why not using 0 directly?

Reply:
SerdeConfigHelper is a struct for holding deserialized values. Obviously, user may omit some of the fields so that the corresponding field in the struct becomes `None`. By using `0`, we cannot distinguish validation errors any more.

Reply:
you mean that we  may have the same problem for all other configurations? Like column family, if we don't set, the default will be 0?

Reply:
We should keep consistency.